go/vt/sqlparser/ast.go

```diff
@@ -1762,11 +1762,12 @@ type CharsetAndCollate struct {
 
 // DBDDL represents a CREATE, DROP database statement.
 type DBDDL struct {
-	Action         string
-	DBName         string
-	IfNotExists    bool
-	IfExists       bool
-	CharsetCollate []*CharsetAndCollate
+	Action           string
+	SchemaOrDatabase string
+	DBName           string
```

Comment:
`DBName` referring to either a schema or database based on `SchemaOrDatabase` is a little confusing. `DBName` implies it's _only_ the database name, so changing this to something else (maybe just `Name`?) seems a bit clearer to me.